package.json

```
@@ -36,6 +36,7 @@
     "diff": "node diff.js",
     "changelog": "node changelog.js",
     "build": "node run.js build all",
+    "test": "yarn test-unit && yarn test-integration && yarn test-integration-once && yarn test-integration-now-dev && yarn test-lint",
```

Let's run `yarn test-lint` first

```suggestion
    "test": "yarn test-lint && yarn test-unit && yarn test-integration && yarn test-integration-once && yarn test-integration-now-dev",
```

---

Ok : )

---

Changes made @styfle  👌